definitions/contested/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -6198,92 +6198,6 @@
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Y"
   },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "FinancialRemedyContested",
-    "CaseEventID": "FR_generalOrderCourtAdmin",
-    "CaseFieldID": "generalOrderCreatedBy",
-    "PageFieldDisplayOrder": 1,
-    "DisplayContext": "MANDATORY",
-    "PageID": 1,
-    "PageDisplayOrder": 1,
-    "PageColumnNumber": 1,
-    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/case-orchestration/documents/preview-general-order",
-    "CallBackURLMidEvent_1": "3,4,5",
-    "ShowSummaryChangeOption": "N"
-  },
```

Comment:
This event you are removing and clubbing another event with this. Are they using same fields?

Reply:
Yes FR_generalOrderCourtAdmin is the same as FR_generalOrder apart from permissions. One was for judges and the other for caseworkers. There is no other difference so has been removed and the permissions amended on FR_generalOrder so that it can be used by both judges and caseworkers.